exercises/I-array-properties/exercise.js

```diff
@@ -6,8 +6,11 @@
 */
 
 function isEmpty(arr) {
-  return; // complete this statement
-}
+ if (arr.length === 0){
```

Comment:
Using a similar coding style to those in **F-predicates**, how can this function be reduced  to  one line?